File: MODUL 2 Praktikum Pemrograman/PRAKTIKUM 2 C/PRAK205_2410817120021_Aulia-Az-Zahra.c
#include <stdio.h>
#include <math.h>

int main() 
{
    int A, B;
    
    scanf("%d", &A); 
    scanf("%d", &B);

    int alas = sqrt(B * B - A * A);

    int keliling = A + B + alas; 
    float luas   = 0.5 * A * alas;

    printf("Alas     = %d cm\n", alas); 
    printf("Tinggi   = %d cm\n", A);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas     = %.f cm^2\n", luas); 

    return 0;
}

File: MODUL 2 Praktikum Pemrograman/PRAKTIKUM 2 PY/PRAK205_2410817120021_Aulia-Az-Zahra.py
import math

sisiTinggi = input()

if len(sisiTinggi.split()) == 2:
    sisiTinggi, sisiMiring = map(int, sisiTinggi.split())
elif len(sisiTinggi.split()) == 1:
    sisiTinggi = int(sisiTinggi)
    sisiMiring = int(input())

alas     = int(math.sqrt(sisiMiring**2 - sisiTinggi**2))
keliling = sisiTinggi + sisiMiring + alas
luas     = int(0.5 * sisiTinggi * alas) 

print(f"Alas     = {alas} cm")
print(f"Tinggi   = {sisiTinggi} cm")
print(f"Keliling = {keliling} cm")
print(f"Luas     = {luas} cm^2")